src/pages/ScholarshipForm.jsx

```diff
@@ -25,11 +33,28 @@ function ScholarshipForm() {
         id={id}
         label={label}
         required
+        fullWidth
         value={formFieldStates[id]}
         onChange={updateFn(id)}
       />
     );
   }
+  function scholarshipTypeField(id, label, enums) {
+    return (
+      <FormControl>
+        <InputLabel shrink>{label}</InputLabel>
+        <Select value={formFieldStates[id]} onChange={updateFn(id)}>
+          {enums.map((option) => {
+            return (
+              <MenuItem key={option} value={option}>
+                {option}
+              </MenuItem>
+            );
+          })}
+        </Select>
+      </FormControl>
+    );
+  }
```

Comment:
This would be a good component to refactor out. Maybe something like `EnumSelectField.jsx`